.gitignore

```diff
@@ -5,3 +5,5 @@ lib
 /npm-packages-offline-cache
 tmp
 yarn
+/package-lock.json
```

Comment:
Yeah, I'm struggling with this one a little.  We can discuss it more when you are out here next week, but the gist is that our plugins are all still on npm, so when linking modules into the root cli project for development we have two different packaging solutions in the linked tree, whether we use npm or yarn.  So either we convert all our plugins to yarn or temporarily link cli-engine into the cli project using npm just for development.